### PR TITLE
Feature/snippet routing

### PR DESCRIPTION
**Pull request by vrael560**
@blimusiek as we spoken before, merging routing to your branch first

### CODE REVIEW

File: server/routes/snippets.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+});
+
+/* GET snippet by id */
+router.get('/:id', function (req, res) {
+  models.Snippet.findById(req.params.id).then(function (s) {
+    res.send(s);
+  });
+});
+
+/* POST new snippet  */
```
Comment (blimusiek) [inline server/routes/snippets.js]: @vrael560 can we change it to ```POST /snippets``` for create?

File: server/routes/snippets.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                language: s.language
+      });
+    });
+
+    res.send(mappedSnippets);
+  });
+});
+
+/* GET snippet by id */
+router.get('/:id', function (req, res) {
```
Comment (blimusiek) [inline server/routes/snippets.js]: @vrael560 and that to ```GET /snippets/:id```

**Comment by blimusiek** [comment]
And can I ask you to check what is wrong with travis build?

File: server/routes/snippets.js (hunk 1 above)
Comment (vrael560) [inline server/routes/snippets.js]: routes are scoped in server/app.js
```
var snippets = require('./routes/snippets');
...
app.use('/snippets', snippets);
```

Comment (blimusiek) [inline server/routes/snippets.js]: oh I miss it

**Comment by blimusiek** [comment]
It's ok for me (@vrael560 just remember add tests to that next time)

**Pull request merged**